spec/spec_helper.rb

```diff
@@ -0,0 +1,92 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
Please add integration tests for the app.

Reply:
@SajjadAhmad14 , I don't see tests as part of the requirements which is why I didn't add too many. However, if I was to add integration tests, do you know how I can bypass the user authentication to test the contents of the page?

Reply:
@TracK92 I noticed that the `Sneak peek: Recipe app` activity is implicit (**refers to something that is implied and not stated directly**) with the requirement for testing. I understand why it can be confusing and seems like it's not part of the requirement, but it is.

On the Technical Setup, you are required to set up RSpec and Capybara testing libraries.
![image](https://user-images.githubusercontent.com/38088818/201319640-07962bc4-38f9-4d7e-b22e-693d8daf9e4a.png)

- This implies that you must write `unit tests using RSpec` and `integration test using Capybara`.

Reply:
Thanks for pointing out these errors @KDlamini  and @SajjadAhmad14 👋 we have added integration tests already.👋